### DISCUSSION TITLE
This is new a feature lol

### DISCUSSION THREAD

**Discussion by Poujhit** [Chennai-Sharks/EMedico#8] [General]
lol nothing.

**Comment by Poujhit** [comment]
This is just like issues but with extra buttons lol

**Comment by Poujhit** [comment]
And can use markdown for discussions, aka chat. **xDD**  and can be edited too. Noice

**Reply by Poujhit** [reply]
Lol and we can see the original messages too. gg

**Reply by Poujhit** [reply]
@NaveenSriramR @NikhileshJr08 , tagging also works xDD. Github discussions is cool.

**Reply by NikhileshJr08** [reply]
Ly thala
